README.md

```diff
@@ -9,6 +9,40 @@
 npm install screwdriver-artifact-bookend
 ```
 
+## Functionality
+
+Store all the artifacts saved in the Artifacts Directory to the build folder in S3, to make sure they're not all erased when the build is complete. This bookend currently acts only as a teardown plugin. All artifacts files/folders are written to S3 on teardown.
```

Comment:
There are a lot of S3 being mentioned with this bookend. If the Store service is supposed to be an abstraction over S3, we should target how it interacts with the Store rather than S3.